superset/views.py

```diff
@@ -1104,6 +1104,34 @@ def ping():
     return "OK"
 
 
+class KV(BaseSupersetView):
+
+    """used for storing and retrieving key value pairs"""
+
+    @log_this
+    @expose("/store/", methods=['POST', 'GET'])
+    def store(self):
+        value = request.form.get('data')
+        base_url = request.form.get('baseUrl') \
+            if request.form.get('baseUrl') else ''
+        obj = models.KeyValue(value=value)
+        db.session.add(obj)
+        db.session.commit()
+        return("http://{request.headers[Host]}/{base_url}?id={obj.id}".format(
```

Comment:
base_url is for generalizing this model. If we directly return /kv/id like we do in shortener, sqllab redux app won't catch it on pre-rendering because url is not marking to /sqllab. Adding base_url here hooks it to the right url with the right redux app we want